src/lib/config.ts

```diff
@@ -28,6 +56,15 @@ export function intVar<R>(varName: string, defaultValue?: R): number | R {
 	return i;
 }
 
+export const openVpnVar = memoizee(
```

Comment:
You don't really need all that -- isn't plain assignment working?

Reply:
I've done it purely to ensure the replace doens't get called every time.

Reply:
Agreed, there's no point invoking the replace more than once, but doesn't the following work as well?

```
export const openVpnVar = openVpnConfig.replace(
	'remote VPN_DETAILS',
	`remote ${requiredVar('VPN_HOST')} ${requiredVar('VPN_PORT')}`,
);
```

Reply:
It does, I've actually assigned it properly now. :)